ui/src/app/views/export-data-set-modal.tsx

```diff
@@ -132,18 +159,29 @@ class ExportDataSetModal extends React.Component<
       }
     });
     return <Modal loading={loading || notebooksLoading}>
-      <ModalTitle>Export {dataSet.name} to Python Notebook</ModalTitle>
+      <ModalTitle>Export {dataSet.name} to Notebook</ModalTitle>
       <ModalBody>
         <Button data-test-id='code-preview-button'
                 onClick={() => this.setState({seePreview: !seePreview})}>
           {seePreview ? 'Hide Preview' : 'See Code Preview'}
         </Button>
         {seePreview && <React.Fragment>
-          {queries.length === 0 && <SpinnerOverlay />}
-          <TextArea disabled={true} onChange={() => {}} style={{marginTop: '1rem'}}
+          {Array.from(queries.values())
+            .filter(query => query !== undefined).length === 0 && <SpinnerOverlay />}
+          <div style={styles.codePreviewSelector}>
+            {Object.keys(KernelTypeEnum).map(kernelTypeEnumKey => KernelTypeEnum[kernelTypeEnumKey])
+              .map((kernelTypeEnum, i) =>
```

Comment:
Nice!  Could we rename this `kernelType` since it's a specific instance?

What is `i` here?  Just an index, or something more meaningful?

Reply:
Just an index.